doc/whats-new.rst

```diff
@@ -36,6 +36,15 @@ Breaking changes
 Deprecations
 ~~~~~~~~~~~~
 
+- Start deprecation cycle for changing the default keyword arguments to ``concat``, ``merge``, ``combine``, ``open_mfdataset``.
+  Emits a ``FutureWarning`` when using old defaults and new defaults would result in different behavior.
+  Adds an option: ``use_new_combine_kwarg_defaults`` to opt in to new defaults immediately.
+  New values are:
+    - ``data_vars``: "minimal"
+    - ``coords``: "minimal"
+    - ``compat``: "override"
+    - ``join``: "exact"
+  By `Julia Signell <https://github.com/jsignell>`_.
```

Comment:
```suggestion

  New values are::
  
    - ``data_vars``: "minimal"
    - ``coords``: "minimal"
    - ``compat``: "override"
    - ``join``: "exact"
  
  By `Julia Signell <https://github.com/jsignell>`_.
```

Reply:
This hopefully fixes the sphinx error on RTD.

Reply:
😆 🤣 well that is a silly one. Thank you for spotting it.

Reply:
Yeah, but for the mypy errors I won't be of any help.